src/components/ExportDataModal.js

```diff
@@ -0,0 +1,228 @@
+import _ from 'lodash/fp'
+import PropTypes from 'prop-types'
+import { b, div, h } from 'react-hyperscript-helpers'
+import { buttonPrimary, spinnerOverlay } from 'src/components/common'
+import ErrorView from 'src/components/ErrorView'
+import { icon } from 'src/components/icons'
+import Modal from 'src/components/Modal'
+import { withWorkspaces, WorkspaceSelector } from 'src/components/workspace-utils'
+import { ajaxCaller } from 'src/libs/ajax'
+import colors from 'src/libs/colors'
+import { reportError } from 'src/libs/error'
+import { requiredFormLabel, formLabel } from 'src/libs/forms'
+import * as Nav from 'src/libs/nav'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+import { Component } from 'src/libs/wrapped-components'
+import validate from 'validate.js'
+
+
+class infoTile extends Component {
+  static propTypes = {
+    infoStyle: PropTypes.object.isRequired,
+    content: PropTypes.string.isRequired,
+    iconName: PropTypes.string.isRequired
+  }
+
+  render() {
+    const { infoStyle, content, iconName } = this.props
+    return div({ style: { ...infoStyle, display: 'flex', alignItems: 'center' } }, [
+      icon(iconName, { size: 36, className: 'is-solid', style: { flex: 'none', marginRight: '0.5rem' } }),
+      content
+    ])
+  }
+}
+
+
+export default _.flow(
+  ajaxCaller,
+  withWorkspaces()
+)(class ExportDataModal extends Component {
+  static propTypes = {
+    onDismiss: PropTypes.func.isRequired,
+    selectedEntities: PropTypes.array.isRequired,
+    selectedDataType: PropTypes.string.isRequired,
+    runningSubmissionsCount: PropTypes.number.isRequired
+  }
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      hardConflicts: [],
+      softConflicts: [],
+      additionalDeletions: [],
+      selectedWorkspaceId: undefined,
+      error: undefined,
+      copying: false,
+      selectedEntityType: props.selectedDataType
```

Comment:
This doesn't have to be in state since it doesn't change, so you could just use the prop instead.